### DISCUSSION TITLE
Has anyone got Composable Lora to work?

### DISCUSSION THREAD

**Discussion by econundrum** [vladmandic/sdnext#1136] [Q&A]
Has anyone managed to get the composable Lora extension to actually work I want to use it with latent couple but all I get with compossible Lora is a error "Expect ll tensors to be on the same device it appears one is on CPU and one on GPU.

Latent couple works ok but I'd like to be able to use different LorAs. I run with --medvram

**Comment by tkiikt** [comment]
#225

**Comment by vladmandic** [comment]
also, <https://github.com/opparco/stable-diffusion-webui-composable-lora/issues/19>

**Comment by econundrum** [comment]
Thank you for the info.

**Comment by tomudo** [comment]
It looks like it broke again because recursive error limit.
